features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/SessionEndedMetricDispatcher.kt

```diff
@@ -17,11 +17,13 @@ internal class SessionEndedMetricDispatcher(private val internalLogger: Internal
 
     private val metricsBySessionId = ConcurrentHashMap<String, SessionEndedMetric>()
 
+    private var lastSessionId: String? = null
```

Comment:
this needs to be `AtomicReference` then, following thread-safe design of this class.

Reply:
This is removed, we don't need to record the last session id